servers/rendering/shader_language.h

```diff
@@ -598,10 +603,21 @@ class ShaderLanguage {
 		};
 
 		struct Varying {
+			enum Stage {
+				STAGE_UNKNOWN,
+				STAGE_VERTEX,
```

Comment:
So vertex and fragment stages only are errors I guess?

Reply:
Yeah, and these are hints to correctly handle this behavior.

Reply:
ok, can you add comments in those two? like "the following two are invalid, they are used as transition states to a valid one, or as error in case of misuse" or something like this.

Reply:
Done